discord/ext/pages/pagination.py

```diff
@@ -166,10 +167,12 @@ async def callback(self, interaction: Optional[discord.Interaction] = None):
         """
 
     def update_files(self) -> Optional[List[discord.File]]:
```

Comment:
While we're at it, let's update the typehint to the new syntax.
```suggestion
    def update_files(self) -> list[discord.File] | None:
```